docs-content/installing.html.md.erb

```diff
@@ -145,8 +144,17 @@ to report the metric the filter entry will be
 For 4.4.x controller, please create an applicaiton on the controller and create a `Go SDK` tier associated with that application and record the tier-id.
 Enter the AppName, Tier, TierID and Node names in the corresponding fields of the **Overriding Defaults** form.
 
+
+## <a id='Scaling'></a> Scalability
+
+Starting 4.6.* version of the tile, AppDynamics Nozzle is completely scalable to any number of desired instances. However,
+it is recommended to scale to the number of traffic controller instances. See the [official cloud foundry recommendation](https://docs.cloudfoundry.org/loggregator/log-ops-guide.html#scaling-nozzles)
```

Comment:
to scale the number of nozzle instances to match the number of traffic controller instances.